### DISCUSSION TITLE
Is there any way to work around the 3-column minimum on dashboards?

### DISCUSSION THREAD

**Discussion by zx48** [datalust/seq-tickets#2216] [General]
Hi all,

I'm finding the 3-column minimum for widgets on dashboards very limiting. I've got a very wide screen and even setting to 1 would make things pretty wide.

So I guess a question for datalust and anyone else that can help - why the 3-col minimum and is there a work around?

Thanks,
CJ

![image](https://github.com/datalust/seq-tickets/assets/1654798/edcf1b0c-7039-4dcb-9d39-f3fd327413a9)

**Comment by nblumhardt** [comment]
Thanks for raising this. I think, historically, the limit was set so that the chart can't be made so narrow that the width controls disappear.

But, this needs an update - I'll raise an issue to track it :+1: 

In the meantime, as a workaround, beside the title of the dashboard there's a dropdown with an "Edit raw JSON" option. Choosing this, you can find the chart and modify its width value directly.

Hope this helps,
Nick
